### PR TITLE
Clarify README reg. async env loading

### PR DESCRIPTION
**Pull request by bbonamin**
Add a paragraph to the README explaining what's written in https://github.com/fastify/fastify-env/issues/108#issuecomment-891747148 to help other newcomers.

I've been bit by this problem too and got confused for quite a while. Finding that issue comment was key, and according to other comments and reactions to it, it seems I'm not the only one. 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,22 @@ const options = {
...
 
+```js
+await fastify.register(fastifyEnv)
+// fastify.config can be used in here
+```
+
+OR
+```js
+fastify.register(fastifyEnv)
+await fastify.after()
```
Comment (mcollina) [inline README.md:78]:
```suggestion
await fastify
```

**Review by mcollina — COMMENTED** [review]
No need to use `after` here

File: README.md [hunk 2]
```diff
@@ -62,7 +62,22 @@ const options = {
 }
 
 ```
+### Using @fastify/env to configure other plugins
```
Comment (Eomm) [inline README.md:65]: I would move it to the `docs/Guides` folder instead

File: README.md (hunk 1 above)
Comment (bbonamin) [reply README.md:78]: Works for me, I was just following the original comment though.

File: README.md (hunk 2 above)
Comment (jsumners) [reply README.md:65]: This is the `@fastify/env` repo, not the core `fastify` repo.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by bbonamin** [comment]
Hey, is there anything else you'd like me to fix before this can be merged? Thanks 😄

**Comment by mcollina** [comment]
thanks! I lost this!

**Pull request merged**